spec/mongoid_spec.rb

```diff
@@ -13,22 +13,51 @@
       end
     end
 
-    context "when a block is supplied" do
+    context "when a block is given" do
 
-      before do
+      after do
         Mongoid.configure do |config|
-          config.preload_models = true
+          config.preload_models = false
```

Comment:
This should use `config_override` to return the value to the previous setting.

Reply:
Done